src/main/java/de/focusshift/zeiterfassung/publicholiday/PublicHoliday.java

```diff
@@ -0,0 +1,29 @@
+package de.focusshift.zeiterfassung.publicholiday;
+
+import java.time.LocalDate;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.function.Function;
+
+/**
+ * Describes a public holiday for the given date.
+ */
+public record PublicHoliday(LocalDate date, Function<Locale, String> description) {
```

Comment:
Es gibt die Möglichkeit, dass zwei Feiertage an einem Tag sind. Wurde das hier bedacht? In der UV haben wir ein Ticket dafür. Das passiert zum Beispiel in Kroatien, wenn ein Feiertag wie Ostern/Pfingsten auf einen fixen Feiertag fällt.

Reply:
das ist im `PublicHolidayCalendar`. Da gibt es für ein Datum eine Menge an Feiertagen.

ist die frage wie das mit der aktuellen `PublicHoliday#equals` Funktion passt. Vielleicht muss da noch ein aufgelöstes `description` mit dazu 🤔